tensorboard/components/tf_dashboard_common/tf-downloader.html

```diff
@@ -89,10 +81,10 @@
         return urlFn(tag, run);
       },
       _csvName(tag, run) {
-        return `run_${run},tag_${tag}.csv`;
+        return `run_${run}-tag-${tag}.csv`;
```

Comment:
this uses an underscore for run but dashes for the rest - make it consistent?  ditto below

Reply:
This was for consistency with the original scalar-card code

<https://github.com/tensorflow/tensorboard/blob/fc61089cefca2c909a5cf7de0fc3ac44e0320720/tensorboard/plugins/scalar/tf_scalar_dashboard/tf-scalar-card.html#L112>

but I’m fine with disregarding that.

Reply:
Ah okay.  I think we can just disregard that; I doubt anyone is depending on the exact format of that downloaded file name.